client/src/partials/exchange/exchange.html

```diff
@@ -63,74 +65,75 @@
-              <label class="control-label col-md-6">{{ ExchangeCtrl.formatCurrency(id) }} </label>
-              <div class="col-md-6">
-                {{ row.rate | currency:id }}
-                <a class="btn btn-warning text-right" id="current-{{ $index + 1 }}" ng-click="ExchangeCtrl.setExchangeRate(id, row)">{{ "FORM.BUTTONS.SUBMIT" | translate }}</a>
-              </div>
-            </div>
-            <div ng-if="!ExchangeCtrl.current">
-              <div  class="form-group" ng-repeat="currency in ExchangeCtrl.outCurrencies">
-                <label class="control-label col-md-6">{{ ExchangeCtrl.formatCurrency(currency.id) }} </label>
-                <div class="col-md-6">
-                  {{ 0 | currency:currency.id }}
-                  <a class="btn btn-warning text-right" id="current-{{ $index + 1 }}" ng-click="ExchangeCtrl.setExchangeRate(currency.id,'')">{{ "FORM.BUTTONS.SUBMIT" | translate }}</a>
-                </div>
-              </div>
-            </div>
-            <span data-role="feedback" ng-switch="ExchangeCtrl.feedback">
-              <span ng-switch-default></span>
-
-              <span ng-switch-when="create_success" id="create_success" class="text-success">
-                <i class="glyphicon glyphicon-ok-sign"></i> {{ "FORM.INFOS.SAVE_SUCCESS" | translate }}
-              </span>
-
-              <span ng-switch-when="update_success" id="update_success" class="text-success">
-                <i class="glyphicon glyphicon-ok-sign"></i> {{ "FORM.INFOS.UPDATE_SUCCESS" | translate }}
-              </span>
-
-              <span ng-switch-when="delete_success" id="delete_success" class="text-success">
-                <i class="glyphicon glyphicon-ok-sign"></i> {{ "FORM.INFOS.DELETE_SUCCESS" | translate }}
-              </span>
 
-              <span ng-switch-when="invalid-date" id="invalid-date" class="text-danger">
-                <i class="glyphicon glyphicon-warning-sign"></i> {{ "FORM.VALIDATIONS.INVALID_DATE" | translate }}
-              </span>
-            </span>
+            <dl class="dl-horizontal">
+              <dt ng-repeat-start="(id, row) in ExchangeCtrl.current">
+                {{ ExchangeCtrl.Currencies.format(id) }}
+              </dt>
+              <dd ng-repeat-end>
+                {{ row.rate | currency:id }}
+                <a class="btn btn-warning text-right" id="current-{{ $index + 1 }}" ng-click="ExchangeCtrl.setExchangeRate(id, row)">
+                  <i class="fa fa-pencil-square-o"></i> {{ "FORM.BUTTONS.EDIT" | translate }}
+                </a>
+              </dd>
+            </dl>
+
+            <dl ng-if="!ExchangeCtrl.current" class="dl-horizontal">
+              <dt ng-repeat-start="currency in ExchangeCtrl.outCurrencies">
+                {{ ExchangeCtrl.Currencies.format(currency.id) }}
+              </dt>
+              <dd ng-repeat-end>
+                {{ 0 | currency:currency.id }}
```

Comment:
Ah, I know what this is.  For rates that are not defined, it templates in 0 automagically.  For example, if we support USD, FC, and EURO, yet only have rates for USD - FC, it would read:
 - 990 FC
 - 0 EURO

... Potentially it should just say _"Rate Not Defined"_ or something.